config/locales/en.yml

```diff
@@ -128,6 +228,20 @@ en:
         part1: "Please %{activate_account_link}."
         part2: "Once your account is up and running, you can post your fanworks, set up email subscriptions to let you know when your favorite creators or works have updated, set preferences to customize the way the site looks and works for you, keep track of the works you've viewed on the Archive via your history, and much more."
         part3: "There's lots of information and advice on how to use the Archive in our %{faq_link}. You'll find the latest news about site developments in our %{admin_posts_link}. If you need more help, run into a bug, or have questions or comments, please %{contact_support_link}, who are always happy to help out."
+    invitation:
+      subject: "%{app_name} Invitation"
```

Comment:
They weren't there before translation, either - do we want to make that text change as well now, or leave it as it currently is?